tinygrad/helpers.py

```diff
@@ -196,15 +195,14 @@ def wrapper(*args, **kwargs) -> bytes:
   return wrapper
 
 # *** http support ***
-
 def fetch(url:str, name:Optional[Union[pathlib.Path, str]]=None, allow_caching=not getenv("DISABLE_HTTP_CACHE")) -> pathlib.Path:
   if url.startswith(("/", ".")): return pathlib.Path(url)
   fp = pathlib.Path(name) if name is not None and (isinstance(name, pathlib.Path) or '/' in name) else pathlib.Path(_cache_dir) / "tinygrad" / "downloads" / (name if name else hashlib.md5(url.encode('utf-8')).hexdigest())  # noqa: E501
   if not fp.is_file() or not allow_caching:
     with urllib.request.urlopen(url, timeout=10) as r:
       assert r.status == 200
       total_length = int(r.headers.get('content-length', 0))
-      progress_bar = tqdm(total=total_length, unit='B', unit_scale=True, desc=url)
+      progress_bar = tinytqdm(total=total_length, unit='B', desc=f"{url}: ", disable=False)
```

Comment:
Does this support unit_scale?

Reply:
it doesn't, unit_scale is the only thing missing from being an exact replacement with pretty much same API (for the tinygrad usecases)
I didn't add it since my solution was already quite a bit over the bounty line count